Description: Fix the plugin name in firefox
 Use a logical plugin name rather than "mplayerplug-in is now gecko-mediaplayer"
Author: Jackson Doak <noskcaj@ubuntu.com>
Bug: https://code.google.com/p/gecko-mediaplayer/issues/detail?id=190
Bug-Ubuntu: https://launchpad.net/bugs/1313163
Forwarded: not-needed

--- gecko-mediaplayer-1.0.9.orig/src/plugin_types_std.cpp
+++ gecko-mediaplayer-1.0.9/src/plugin_types_std.cpp
@@ -142,7 +142,7 @@ NPError PluginGetValue(NPPVariable varia
     // some sites use this description to figure out what formats can be played. So we have to make sure the 
     // description matches the features
     if (variable == NPPVpluginNameString) {
-        *((const char **) value) = "mplayerplug-in is now gecko-mediaplayer " VERSION;
+        *((const char **) value) = "Gecko Media Player" VERSION;
     }
     if (variable == NPPVpluginDescriptionString) {
         *((const char **) value) =
